_data/locales/tr.yml

```diff
@@ -27,4 +27,5 @@ tr:
       analytics: Analytics Verileri
     foot:
       code: Özgün kodu yazan <a href="https://mxcl.github.io/">Max HowelL</a>.
-      page: Siteyi hazırlayanlar <a href="https://www.exomel.com/">Rémi Prévost</a>, <a href="https://mikemcquaid.com">Mike McQuaid</a>, <a href="https://danilalo.com/">Danielle Lalonde</a> ve <a href="https://sadikkuzu.com">Sadık Kuzu</a>.
+      page: Siteyi hazırlayanlar <a href="https://exomel.com/">Rémi Prévost</a>, <a href="https://mikemcquaid.com">Mike McQuaid</a>, <a href="https://danilalo.com/">Danielle Lalonde</a>.
+      translation: Tercüme eden <a href="https://sadikkuzu.com">Sadık Kuzu</a>.
```

Comment:
@sadikkuzu This is what Google Translate gave me for "Translated by"

Reply:
Sure @EricFromCanada , thanks.